clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -58,8 +60,13 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
     def region = description.region
     def serverGroupName = description.serverGroupName
     def serverGroup = GCEUtil.queryServerGroup(googleClusterProvider, accountName, region, serverGroupName)
+    def isRegional = serverGroup.regional
+    // Will return null if this is a zonal server group.
     def zone = serverGroup.zone
-    def managedInstanceGroup = GCEUtil.queryManagedInstanceGroup(project, zone, serverGroupName, credentials);
+    def managedInstanceGroup =
+      isRegional
+      ? GCEUtil.queryRegionalManagedInstanceGroup(project, region, serverGroupName, credentials)
+      : GCEUtil.queryZonalManagedInstanceGroup(project, zone, serverGroupName, credentials)
```

Comment:
FYI: Intellij marks this style as an error.  For example: 
![screenshot from 2016-05-26 09 41 40](https://cloud.githubusercontent.com/assets/13141550/15576468/13adb184-2326-11e6-8ea1-55809393a06d.png)